### PR TITLE
Implement unified navbar JSP

### PR DESCRIPTION
**Pull request by ksingarajah**
Visitor:
screen shot 2018-05-30 at 11 40 23 pm

User (non-admin):
screen shot 2018-05-30 at 11 42 45 pm

Admin:
screen shot 2018-05-30 at 11 45 00 pm

### CODE REVIEW

**Review by ricglz — APPROVED** [review]
Love your comments in navbar.jsp, thank you for doing it.

File: src/main/webapp/WEB-INF/view/navbar.jsp [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    General navigation bar setup (3 main "modes")
+    - Visitor (not logged in)
+        - Have access to: Register, Login, About
+    - User (logged in)
+        - Have access to: Activity Feed, Profile, Conversations, About
+        - Display "Hello, <user>!"
+    - Admin (logged in + admin)
+        - Have access to: Activity Feed, Admin Page, Profile, Conversations, About
+        - Display "Hello, Admin <user>!"
+-->
```
Comment (manjilgtm) [inline src/main/webapp/WEB-INF/view/navbar.jsp:11]: This was very helpful while reviewing your code.

**Comment by manjilgtm** [comment]
Everything looks good!

**Comment by tsalako** [comment]
Yea. Looks good. A quick rule of thumb moving forward though. We should let the owner of the pull request merge to master. They may still want to add a bit more/make changes before merging.

**Pull request merged**